pkgs/applications/misc/sigal/default.nix

```diff
@@ -1,14 +1,16 @@
-{ lib, python3Packages, ffmpeg_3 }:
+{ lib, python3Packages, pythonAtLeast, ffmpeg }:
```

Comment:
You're still referring to `ffmpeg_3` down in `makeWrapperArgs` which is causing this to fail.

Also you don't need the `pythonAtLeast` arg here anymore.